lib/drivers/veres.js

```diff
@@ -596,6 +596,10 @@ function _delay(delay) {
 api.info = async options => {
   _verbose(options, 'DID:', options.did);
 
+  // check for optional callback param (used to return the results directly
+  //  instead of displaying them)
+  const callback = options.callback;
```

Comment:
good place for destructuring assignment:
```
const {callback} = options;
```

Reply:
Thanks! Will use destructuring instead.